### PR TITLE
Update Brave and Gradle

### PR DESCRIPTION
**Pull request by beckje01**
Switch to using Gradle 4.6 BOM

Update to Brave 4.18.2

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,24 +35,26 @@ repositories {
...
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.6' // required for coverage
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
-    testCompile 'cglib:cglib-nodep:3.2.1'
-    testCompile 'org.objenesis:objenesis:2.2'
-    testCompile 'com.nagternal:spock-genesis:0.4.0'
-    testCompile 'org.slf4j:slf4j-nop:1.7.21'
-    testCompile 'io.ratpack:ratpack-groovy-test:1.4.1'
-    testCompile "io.zipkin.brave:brave-instrumentation-http-tests:${braveVersion}"
-    testCompile 'com.squareup.okhttp3:mockwebserver:3.9.1'
+    testImplementation 'io.zipkin.reporter:zipkin-reporter'
```
Comment (codefromthecrypt) [inline build.gradle:46]: nit I might sort these somehow.. makes it more clear the zipkin ones are missing versions

**Review by codefromthecrypt — COMMENTED** [review]
looks good

**Review by rappleg — APPROVED** [review]

**Review by mrobinet — APPROVED** [review]

**Review by llinder — APPROVED** [review]

**Comment by hyleung** [comment]
Thanks, everyone!

File: build.gradle (hunk 1 above)
Comment (tylerbenson) [reply build.gradle]: not sure this is worth the effort.  skipping for now.

**Pull request merged**